canonical-web-and-design/ubuntu.com
# convert engage entries to articles

## Done

- converted engage index entry elements to `article`s

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page layout isn't broken


## Issue / Card

Fixes #6295
